mitodl/ol-infrastructure
# fix: update config to avoid logo leak to production

### What are the relevant tickets?
None, 

### Description (What does it do?)
- We recently faced issues when we were updating the logos and a logo leaked out into all production MFEs because we were using GitHub-based logos. So, as soon as we updated the files on GitHub the logos got updates on production unexpectedly.

We haven't yet updated the logos for the MITx Online edX instance but this PR will save the logos from leaking into production when we update the logos.

### How can this be tested?
- Nothing should change visibally. The logos should still look the same. Just they should be loading from self-hosted URLs

**NOTE:** We will eventually need to change the `.png` to `.svg` when we have them in the theme.
